### PR TITLE
Added support for interest, deposit and withdrawal history and start_ts/end_ts to orders-history

### PR DESCRIPTION
**Pull request by financelurker**
see https://exchange-docs.crypto.com/spot/index.html#private-margin-get-interest-history for interest history
see https://exchange-docs.crypto.com/spot/index.html#private-get-deposit-history for deposit history
see https://exchange-docs.crypto.com/spot/index.html#private-get-withdrawal-history for withdrawal history

### CODE REVIEW

**Comment by financelurker** [comment]
@mortyspace what's up with the azure devops builds? This is not just my PR, every build is red over there...

**Comment by mortyspace** [comment]
will fix CI today, the problem with cc reporter

File: src/cryptocom/exchange/structs.py [hunk 1]
```diff
@@ -339,3 +339,74 @@ def create_from_api(
...
+            update_time=data['update_time'],
+            amount=float(data['amount']),
+            address=data['address'],
+            status=data['status']
+        )
+
+
+@dataclass
+class Deposit:
+    currency: str
```
Comment (mortyspace) [inline src/cryptocom/exchange/structs.py:392]: use coin instead and Coin struct

Comment (financelurker) [reply src/cryptocom/exchange/structs.py]: changed as requested, see the last commit

**Comment by financelurker** [comment]
I've also added start_ts and end_ts params for the get_orders_history call.
Is there anything I can help you with merging this PR?

File: src/cryptocom/exchange/private.py [hunk 2]
```diff
@@ -36,13 +37,77 @@ async def get_balance(self) -> Dict[Coin, Balance]:
             for bal in data['accounts']
         }
 
+    async def get_deposit_history(
+            self, currency: str, start_ts: int, end_ts: int, status: int, page: int = 0, page_size: int = 20)\
```
Comment (mortyspace) [inline src/cryptocom/exchange/private.py:41]: here Coin too, after this I will merge ASAP

**Comment by financelurker** [comment]
done, have found "currency" param in get_withdrawal_history, get_deposit_history and get_interest_history.

**Pull request merged**